homework/01LessonOne.js

```diff
@@ -9,34 +9,37 @@ var assert = require('assert');
 // Problem 0:
```

Comment:
Hey there... My bad. I realized I never finished this Here you go & if it's too late that's okay, still wanted to finish it.